CS410_Assignment2_CodeReview/src/sensormain/SensorApplication.java

```diff
@@ -0,0 +1,205 @@
+/*
+ * Lauren Kearney
+ * CS 410
+ * 
+ * three panel GUI for Sensor application
+ * 
+ * */
+
+package sensormain;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import java.util.ArrayList;
+import java.util.List;
+
+import java.awt.*;
+import javax.swing.*;
+
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JProgressBar;
+import javax.swing.border.TitledBorder;
+
+import sensor.PressureSensor;
+import sensor.RadiationSensor;
+import sensor.TemperatureSensor;
+
+public class SensorApplication extends JFrame {
+	
+	public PressureSensorAdapter pressure = new PressureSensorAdapter(new PressureSensor());
+	public RadiationSensorAdapter radiation = new RadiationSensorAdapter(new RadiationSensor());
+	public TemperatureSensorAdapter temperature = new TemperatureSensorAdapter(new TemperatureSensor());
+	
+	public SensorApplication() {
+		
+		int pressureLength = (int) (pressure.value());
+		pressureLength *= 10;
+		int radiationLength = (int) (radiation.value());
+		radiationLength *= 10;
+		int temperatureLength = (int) (temperature.value());
+		temperatureLength /= 4;
+		
+		
+		Dimension dimPressure = new Dimension();
+		dimPressure.height = 50;
+		dimPressure.width = pressureLength;
+		
+		Dimension dimRadiation = new Dimension();
+		dimRadiation.height = 50;
+		dimRadiation.width = radiationLength;
+		
+		Dimension dimTemperature = new Dimension();
+		dimTemperature.height = 50;
+		dimTemperature.width = temperatureLength;
+		
+		Dimension dimMax = new Dimension();
+		dimMax.height = 50;
+		dimMax.width = 100;
+		
+		setTitle("Sensor Tracker");
+		setLayout(new GridLayout(3,1));
+		
+	//Pressure Panel
+		JPanel pressurePnl = new JPanel(new FlowLayout(FlowLayout.LEADING));
+		pressurePnl.setBorder(new TitledBorder("Pressure Sensor"));
+		
+		if (pressure.status().toLowerCase().equals("ok")) {
+			GreenTracker green = new GreenTracker();
+			green.setPreferredSize(dimPressure);
+			pressurePnl.add(green);
+		}
+		else if (pressure.status().toLowerCase().equals("critical")) {
+			YellowTracker yellow = new YellowTracker();
+			yellow.setPreferredSize(dimPressure);
+			pressurePnl.add(yellow);
+		}
+		else if (pressure.status().toLowerCase().equals("danger")) {
+			RedTracker red = new RedTracker();
+			red.setPreferredSize(dimPressure);
+			pressurePnl.add(red);
+		} else {	//Fail safe
+			BlueTracker blue = new BlueTracker();
+			blue.setPreferredSize(dimMax);
+			pressurePnl.add(blue);
+		}
+		
+		JLabel pressureLabel = new JLabel((pressure.status()+ "---->" + pressure.value()), JLabel.RIGHT);

```

Comment:
Now when `pressureValue` is used it is consistent. 
`presure.status()` should also function correctly now that pressure.value() was called.